src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts

```diff
@@ -797,28 +791,19 @@ export class GettingStartedPage extends EditorPane {
 		const someStepsComplete = this.gettingStartedCategories.some(categry => categry.content.type === 'steps' && categry.content.stepsComplete);
 		if (!someStepsComplete && !this.hasScrolledToFirstCategory) {
 
-			const fistContentBehaviour =
-				!this.storageService.get(lastSessionDateStorageKey, StorageScope.GLOBAL) // isNewUser ?
-					? 'openToFirstCategory'
-					: await Promise.race([
-						this.tasExperimentService?.getTreatment<'index' | 'openToFirstCategory'>('GettingStartedFirstContent'),
-						new Promise<'index'>(resolve => setTimeout(() => resolve('index'), 1000)),
-					]);
-
-			if (this.gettingStartedCategories.some(category => category.content.type === 'steps' && category.content.stepsComplete)) {
```

Comment:
This was to double check nothing got checked off while waiting for the exp service, now we're fully sync so the check from 797 won't go stale.